tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpv.py

```diff
@@ -1125,3 +1125,22 @@ def test_rjumpv_backwards_reference_only(
         data=container,
         expect_exception=EOFException.UNREACHABLE_INSTRUCTIONS,
     )
+
+
+def test_rjumpv_at_the_end(
+    eof_test: EOFTestFiller,
+):
+    """
+    https://github.com/ipsilon/eof/blob/main/spec/eof.md#stack-validation 4.i:
+    This implies that the last instruction may be a terminating instruction or RJUMP
+    """
+    eof_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH1(0) + Op.PUSH1(0) + Op.RJUMPI[1] + Op.STOP + Op.RJUMPV[-3],
```

Comment:
```suggestion
                    code=Op.PUSH1(0) + Op.PUSH1(0) + Op.RJUMPI[1] + Op.STOP + Op.RJUMPV[-7](1),
```
Because I think `Op.RJUMPV[-3]` jumps back into the data portion of `RJUMPV` but then `Op.RJUMPV[-4]` would create a stack-underflow, so `Op.RJUMPV[-7](1)` jumps back into `Op.STOP`.

We should get the same `expect_exception=EOFException.MISSING_STOP_OPCODE,` here with this change.

Reply:
could we vmtrace that?